src/Components/OATGraphViewer/OATGraphViewer.tsx

```diff
@@ -115,22 +115,33 @@ const OATGraphViewer = ({ state, dispatch }: OATGraphProps) => {
                 input.contents.forEach((content) => {
                     switch (content['@type']) {
                         case OATComponentHandleName:
-                            addComponentRelationship(
-                                input['@id'],
-                                content,
-                                models.find(
-                                    (model) => model['@id'] === content.schema
-                                ).displayName,
-                                elements
+                            // eslint-disable-next-line no-case-declarations
```

Comment:
This rule does not allow declaring variables right after the `case:` scope. As in
` case first:
const number = 32;
break;
`.

However, it is useful for us to declare in this case since, `foundComponentTarget`, will only be of use under the `OATComponentHandleName` case.

Reply:
Got it. From the docs, it looks like we can use them if we wrap the case body in brackets? https://eslint.org/docs/latest/rules/no-case-declarations